proto/common/common.proto

```diff
@@ -57,6 +57,14 @@ message Destination {
   string path = 2;
 }
 
+message EosCtx {
```

Comment:
it's still my opinion that it's premature to treat this as common. not a blocker though.

Reply:
@olix0r i'll go ahead and create a duplicate `EosCtx`, but note that one copy will still need to live in common, as that's where `TapEvent` is defined.